requirements.txt

```diff
@@ -7,4 +7,5 @@ tqdm >= 4.46.0
 joblib >= 0.14.1
 ipython >= 7.12.0
 pandas
-swifter >= 1.0.3
\ No newline at end of file
+swifter >= 1.0.3
+en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.3.0/en_core_web_sm-2.3.0.tar.gz
```

Comment:
👍 

what about `spacey`? would this install `spacey` automatically as well? Is it needed for this to work?

Reply:
Spacy will automatically be handled.